cfpb/cfgov-refresh
# Fix for ie8 Modernizr error

Fix for ie8 Modernizr error. When we upgraded to `gulp-uglify 2.0` it introduced a bug related to property compression. 

## Changes

- Modified `gulp/tasks/scripts.js` to turn off property compression.

## Testing

- Run `gulp build`
- Visit the site in `ie8`  on sauce or your VM. 
- Notice that there are no ghosts in the console.


## Checklist
